inst/qml/doeFactorial.qml

```diff
@@ -18,206 +18,206 @@ import QtQuick.Layouts 							1.3
 import JASP.Controls 							1.0
 import JASP.Widgets 							1.0
 
-Form 
+Form
 {
-	usesJaspResults:							true
-	columns:									2
-
-	GroupBox
-	{
-		title: 									qsTr("Factor Info")
-		name:									"factorInfo"
-
-		IntegerField
-		{
-			id:									numberOfFactors
-			name:								"numberOfFactors"
-			label:								qsTr("Number of factors")
-			defaultValue:						2
-			min:								2
-			max:								256
-		}
-
-		DropDown
-		{	
-		id:										numberOfLevels
-		name: 									"numberOfLevels"
-		label: 									qsTr("Number of factor levels")
-		indexDefaultValue: 						0
-		values:
-		[
-			{ value: "2", label: qsTr("2")},
-            { value: "3", label: qsTr("3")},
-            { value: "Mixed", label: qsTr("Mixed")}
-		]
-		}
-	}
+    usesJaspResults:							true
+    columns:									2
```

Comment:
1. If you add `debug: true` or `visible: false` you don't need to comment out all of the code.

2. Please use tabs, not spaces. See https://doc.qt.io/qtcreator/creator-indenting-code.html#indenting-qml-files for how to change the settings in Qt.

Reply:
done!